ONSdigital/sdx-console
# Transform testform

**Changes**
Added 'XML' tab to console

**How to test**

Use console app locally in full app setup and check xml can be displayed 
Requires example data in sdx-transform-testform README
Requires sdx-transform-testform; sdx-downstream

**Who can test**

Anyone but @jjeane
